docs/examples/viz_pbr_interactive.py

```diff
@@ -0,0 +1,228 @@
+from fury import actor, material, ui, window
```

Comment:
cool this tutorial but needs more explanation

Reply:
I'm on it. I will push that commit in a few minutes.

Reply:
Done! PTAL.